src/Microsoft.DotNet.Darc/src/Darc/Operations/AddChannelOperation.cs

```diff
@@ -80,5 +80,12 @@ public override async Task<int> ExecuteAsync()
                 return Constants.ErrorCode;
             }
         }
+
+        protected override bool IsOutputFormatSupported(DarcOutputType outputFormat)
+            => outputFormat switch
+            {
+                DarcOutputType.json => true,
+                _ => base.IsOutputFormatSupported(DarcOutputType.text),
```

Comment:
This doesn't seem correct.  Why are we passing "text" to the base class? Since the base Operation class returns "true" for "text", doesn't that make it so that all these things ALWAYS return true?

Reply:
Bah... Yes, I think it should be 
```suggestion
                _ => base.IsOutputFormatSupported(outputFormat),
```